CRITICAL - 2023-04-11 00:24:56 --> Call to undefined method CodeIgniter\Session\Session::unset()
in APPPATH/Controllers/Inscription.php on line 654.
 1 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->sessionLogin()
 2 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 3 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 06:39:24 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php(174): mysqli->real_connect('195.179.239.93', 'root', 'Cityeduc2023', 'u218692649_city...', 3306, '', 0)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#6 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-04-11 06:39:24 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'154.126.98.145' (using password: YES)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_utilisateur`
WHERE `email` = :email:', [...], false)
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 06:41:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php(174): mysqli->real_connect('localhost', 'root', 'Cityeduc2023', 'u218692649_city...', 3306, '', 0)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#6 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-04-11 06:41:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_utilisateur`
WHERE `email` = :email:', [...], false)
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 06:54:23 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo failed: Échec temporaire dans la résolution du nom in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', '/opt/lampp/htdo...', 174, Array)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php(174): mysqli->real_connect('https://cite-ed...', 'root', 'Cityeduc2023', 'u218692649_city...', 3306, '', 0)
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#4 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#9 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#10 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo failed: Échec temporaire dans la résolution du nom in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php(174): mysqli->real_connect('https://cite-ed...', 'root', 'Cityeduc2023', 'u218692649_city...', 3306, '', 0)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo failed: Échec temporaire dans la résolution du nom in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#6 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-04-11 06:54:23 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo failed: Échec temporaire dans la résolution du nom
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_utilisateur`
WHERE `email` = :email:', [...], false)
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 07:02:14 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u218692649_root'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php(174): mysqli->real_connect('195.179.239.93', 'u218692649_root', 'Cityeduc2023', 'u218692649_city...', 3306, '', 0)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#6 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-04-11 07:02:14 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'154.126.98.145' (using password: YES)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_utilisateur`
WHERE `email` = :email:', [...], false)
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 07:05:08 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u218692649_root'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php(174): mysqli->real_connect('myprod.online', 'u218692649_root', 'Cityeduc2023', 'u218692649_city...', 3306, '', 0)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#6 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-04-11 07:05:08 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'154.126.98.145' (using password: YES)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_utilisateur`
WHERE `email` = :email:', [...], false)
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 07:05:45 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u218692649_root'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php(174): mysqli->real_connect('myprod.online', 'u218692649_root', 'Cityeduc2023', 'u218692649_city...', 3306, '', 0)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#6 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-04-11 07:05:45 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'154.126.98.145' (using password: YES)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_utilisateur`
WHERE `email` = :email:', [...], false)
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 07:06:36 --> Error connecting to the database: mysqli_sql_exception: Connexion refusée in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php(174): mysqli->real_connect('154.126.98.145', 'u218692649_root', 'Cityeduc2023', 'u218692649_city...', 3306, '', 0)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connexion refusée in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#6 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-04-11 07:06:36 --> Unable to connect to the database.
Main connection [MySQLi]: Connexion refusée
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_utilisateur`
WHERE `email` = :email:', [...], false)
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 07:06:54 --> Error connecting to the database: mysqli_sql_exception: Connexion refusée in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php(174): mysqli->real_connect('154.126.98.145', 'root', 'Cityeduc2023', 'u218692649_city...', 3306, '', 0)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connexion refusée in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#6 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-04-11 07:06:54 --> Unable to connect to the database.
Main connection [MySQLi]: Connexion refusée
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_utilisateur`
WHERE `email` = :email:', [...], false)
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 07:08:41 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u218692649_root'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php(174): mysqli->real_connect('195.179.239.51', 'u218692649_root', 'Cityeduc2023', 'u218692649_city...', 3306, '', 0)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#6 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-04-11 07:08:41 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'154.126.98.145' (using password: YES)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_utilisateur`
WHERE `email` = :email:', [...], false)
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 07:11:04 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u218692649_root'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php(174): mysqli->real_connect('195.179.239.51', 'u218692649_root', 'Cityeduc2023', 'u218692649_city...', 3306, '', 0)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#6 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-04-11 07:11:04 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'154.126.98.145' (using password: YES)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_utilisateur`
WHERE `email` = :email:', [...], false)
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 07:11:35 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u218692649_root'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php(174): mysqli->real_connect('195.179.239.51', 'u218692649_root', 'Cityeduc2023', 'u218692649_city...', 3306, '', 0)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'154.126.98.145' (using password: YES) in /opt/lampp/htdocs/cite-educative_dev/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /opt/lampp/htdocs/cite-educative_dev/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /opt/lampp/htdocs/cite-educative_dev/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /opt/lampp/htdocs/cite-educative_dev/system/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 /opt/lampp/htdocs/cite-educative_dev/app/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
#6 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
#7 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#8 /opt/lampp/htdocs/cite-educative_dev/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /opt/lampp/htdocs/cite-educative_dev/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-04-11 07:11:35 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'154.126.98.145' (using password: YES)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_utilisateur`
WHERE `email` = :email:', [...], false)
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/Inscription.php(63): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->saveUtilisateur()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 12:31:56 --> mysqli_sql_exception: Table 'u218692649_cityeduc.cs_categorie' doesn't exist in /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/MySQLi/Connection.php:295
Stack trace:
#0 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/MySQLi/Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Model.php(203): CodeIgniter\Database\BaseBuilder->get()
#5 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/BaseModel.php(557): CodeIgniter\Model->doFind(false, NULL)
#6 /home/u218692649/domains/myprod.online/public_html/cite-educative/app/Controllers/Inscription.php(306): CodeIgniter\BaseModel->find()
#7 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(934): App\Controllers\Inscription->inscriptionStructureCategorieZero()
#8 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /home/u218692649/domains/myprod.online/public_html/cite-educative/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-04-11 12:31:56 --> Table 'u218692649_cityeduc.cs_categorie' doesn't exist
in SYSTEMPATH/Database/BaseConnection.php on line 645.
 1 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `cs_categorie`', [], false)
 2 SYSTEMPATH/Model.php(203): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH/BaseModel.php(557): CodeIgniter\Model->doFind(false, null)
 4 APPPATH/Controllers/Inscription.php(306): CodeIgniter\BaseModel->find()
 5 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->inscriptionStructureCategorieZero()
 6 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 7 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 12:32:06 --> mysqli_sql_exception: Table 'u218692649_cityeduc.cs_categorie' doesn't exist in /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/MySQLi/Connection.php:295
Stack trace:
#0 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/MySQLi/Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Model.php(203): CodeIgniter\Database\BaseBuilder->get()
#5 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/BaseModel.php(557): CodeIgniter\Model->doFind(false, NULL)
#6 /home/u218692649/domains/myprod.online/public_html/cite-educative/app/Controllers/Inscription.php(306): CodeIgniter\BaseModel->find()
#7 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(934): App\Controllers\Inscription->inscriptionStructureCategorieZero()
#8 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /home/u218692649/domains/myprod.online/public_html/cite-educative/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-04-11 12:32:06 --> Table 'u218692649_cityeduc.cs_categorie' doesn't exist
in SYSTEMPATH/Database/BaseConnection.php on line 645.
 1 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `cs_categorie`', [], false)
 2 SYSTEMPATH/Model.php(203): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH/BaseModel.php(557): CodeIgniter\Model->doFind(false, null)
 4 APPPATH/Controllers/Inscription.php(306): CodeIgniter\BaseModel->find()
 5 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->inscriptionStructureCategorieZero()
 6 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 7 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 12:32:14 --> mysqli_sql_exception: Table 'u218692649_cityeduc.cs_categorie' doesn't exist in /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/MySQLi/Connection.php:295
Stack trace:
#0 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/MySQLi/Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Model.php(203): CodeIgniter\Database\BaseBuilder->get()
#5 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/BaseModel.php(557): CodeIgniter\Model->doFind(false, NULL)
#6 /home/u218692649/domains/myprod.online/public_html/cite-educative/app/Controllers/Inscription.php(306): CodeIgniter\BaseModel->find()
#7 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(934): App\Controllers\Inscription->inscriptionStructureCategorieZero()
#8 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#9 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /home/u218692649/domains/myprod.online/public_html/cite-educative/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-04-11 12:32:14 --> Table 'u218692649_cityeduc.cs_categorie' doesn't exist
in SYSTEMPATH/Database/BaseConnection.php on line 645.
 1 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `cs_categorie`', [], false)
 2 SYSTEMPATH/Model.php(203): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH/BaseModel.php(557): CodeIgniter\Model->doFind(false, null)
 4 APPPATH/Controllers/Inscription.php(306): CodeIgniter\BaseModel->find()
 5 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->inscriptionStructureCategorieZero()
 6 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 7 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 12:43:27 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`u218692649_cityeduc`.`c_utilisateur_champ_action`, CONSTRAINT `c_utilisateur_champ_action_ibfk_1` FOREIGN KEY (`id_utilisateur`) REFERENCES `c_utilisateur` (`id`)) in /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/MySQLi/Connection.php:295
Stack trace:
#0 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/MySQLi/Connection.php(295): mysqli->query('INSERT INTO `c_...', 0)
#1 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `c_...')
#2 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `c_...')
#3 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseBuilder.php(1801): CodeIgniter\Database\BaseConnection->query('INSERT INTO `c_...', NULL, false)
#4 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseBuilder.php(2192): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Model.php(364): CodeIgniter\Database\BaseBuilder->insertBatch(Array, NULL, 100)
#6 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/BaseModel.php(874): CodeIgniter\Model->doInsertBatch(Array, NULL, 100, false)
#7 /home/u218692649/domains/myprod.online/public_html/cite-educative/app/Controllers/Inscription.php(374): CodeIgniter\BaseModel->insertBatch(Array)
#8 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(934): App\Controllers\Inscription->inscriptionStructureChampActionSave()
#9 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#10 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/u218692649/domains/myprod.online/public_html/cite-educative/public/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-04-11 12:43:27 --> Cannot add or update a child row: a foreign key constraint fails (`u218692649_cityeduc`.`c_utilisateur_champ_action`, CONSTRAINT `c_utilisateur_champ_action_ibfk_1` FOREIGN KEY (`id_utilisateur`) REFERENCES `c_utilisateur` (`id`))
in SYSTEMPATH/Database/BaseConnection.php on line 645.
 1 SYSTEMPATH/Database/BaseBuilder.php(1801): CodeIgniter\Database\BaseConnection->query('INSERT INTO `c_utilisateur_champ_action` (`id_champ_action`, `id_utilisateur`)
VALUES (4,3)', null, false)
 2 SYSTEMPATH/Database/BaseBuilder.php(2192): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
 3 SYSTEMPATH/Model.php(364): CodeIgniter\Database\BaseBuilder->insertBatch([...], null, 100)
 4 SYSTEMPATH/BaseModel.php(874): CodeIgniter\Model->doInsertBatch([...], null, 100, false)
 5 APPPATH/Controllers/Inscription.php(374): CodeIgniter\BaseModel->insertBatch([...])
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->inscriptionStructureChampActionSave()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-11 12:43:31 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`u218692649_cityeduc`.`c_utilisateur_champ_action`, CONSTRAINT `c_utilisateur_champ_action_ibfk_1` FOREIGN KEY (`id_utilisateur`) REFERENCES `c_utilisateur` (`id`)) in /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/MySQLi/Connection.php:295
Stack trace:
#0 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/MySQLi/Connection.php(295): mysqli->query('INSERT INTO `c_...', 0)
#1 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `c_...')
#2 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `c_...')
#3 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseBuilder.php(1801): CodeIgniter\Database\BaseConnection->query('INSERT INTO `c_...', NULL, false)
#4 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Database/BaseBuilder.php(2192): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/Model.php(364): CodeIgniter\Database\BaseBuilder->insertBatch(Array, NULL, 100)
#6 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/BaseModel.php(874): CodeIgniter\Model->doInsertBatch(Array, NULL, 100, false)
#7 /home/u218692649/domains/myprod.online/public_html/cite-educative/app/Controllers/Inscription.php(374): CodeIgniter\BaseModel->insertBatch(Array)
#8 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(934): App\Controllers\Inscription->inscriptionStructureChampActionSave()
#9 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
#10 /home/u218692649/domains/myprod.online/public_html/cite-educative/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/u218692649/domains/myprod.online/public_html/cite-educative/public/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-04-11 12:43:31 --> Cannot add or update a child row: a foreign key constraint fails (`u218692649_cityeduc`.`c_utilisateur_champ_action`, CONSTRAINT `c_utilisateur_champ_action_ibfk_1` FOREIGN KEY (`id_utilisateur`) REFERENCES `c_utilisateur` (`id`))
in SYSTEMPATH/Database/BaseConnection.php on line 645.
 1 SYSTEMPATH/Database/BaseBuilder.php(1801): CodeIgniter\Database\BaseConnection->query('INSERT INTO `c_utilisateur_champ_action` (`id_champ_action`, `id_utilisateur`)
VALUES (4,3)', null, false)
 2 SYSTEMPATH/Database/BaseBuilder.php(2192): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
 3 SYSTEMPATH/Model.php(364): CodeIgniter\Database\BaseBuilder->insertBatch([...], null, 100)
 4 SYSTEMPATH/BaseModel.php(874): CodeIgniter\Model->doInsertBatch([...], null, 100, false)
 5 APPPATH/Controllers/Inscription.php(374): CodeIgniter\BaseModel->insertBatch([...])
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Inscription->inscriptionStructureChampActionSave()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inscription))
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
